### PR TITLE
Move CDL blocks to obsolete package

### LINKED ISSUE #2134: Move CDL blocks to obsolete
This issue is the move these two blocks to the `Obsolete` package:

1) `Continuous.ChangeSign`: This is nowhere used, and can easily be implemented by using a gain of -1.

2) `Continuous.HysteresisWithHold`: This is used only in one model, and can easily be assembled by connecting a hysteresis block with a `TrueFalseHold`.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #2134 and includes all changes from https://github.com/lbl-srg/modelica-buildings/pull/2155 (to avoid merge conflicts).

### CODE REVIEW

File: Buildings/Controls/OBC/CDL/Psychrometrics/DewPoint_TDryBulPhi.mo [hunk 1]
```diff
@@ -52,6 +47,12 @@ ASHRAE Handbook Fundamentals, p. 6.9.
 </html>", revisions="<html>
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/CDL/Psychrometrics/DewPoint_TDryBulPhi.mo:47]: Can we update the reference as `"It is the correlation from 2009 ASHRAE Handbook Fundamentals, p. 1.9, equation 39."`?

Comment (mwetter) [reply Buildings/Controls/OBC/CDL/Psychrometrics/DewPoint_TDryBulPhi.mo:47]: Yes, please add this. I was wondering what the source is as I don't have the Handbook at home.

File: Buildings/Controls/OBC/CDL/Psychrometrics/SpecificEnthalpy_TDryBulPhi.mo [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    defaultComponentName="ent",
+    Documentation(info="<html>
+<p>
+This block computes the specific enthalpy for a given dry bulb temperature, relative air humidity
+and atmospheric pressure.
+The specific enthalpy is zero if the temperature is <i>0</i>&deg;C and if there
+is no moisture.
+</p>
+<p>
+The correlation used in this model is from the 2005 ASHRAE Handbook Fundamentals, p. 6.9.
```
Comment (JayHuLBL) [inline Buildings/Controls/OBC/CDL/Psychrometrics/SpecificEnthalpy_TDryBulPhi.mo:48]: Could we update the reference to `2009 ASHRAE Handbook Fundamentals, p. 1.9, equation 32.`?

**Comment by JayHuLBL** [comment]
Ok, I will update both sources.

**Comment by JayHuLBL** [comment]
@mwetter I have changed the variable name and have no further comment.

**Review by JayHuLBL — APPROVED** [review]

**Pull request merged**